hs_discover/src/components/Search.vue

```diff
@@ -32,17 +28,20 @@
                             </h4>
                         </div>
                         <div id="dateselectors" class="facet-list panel-collapse collapse in" aria-labelledby="headingDate">
-                            <div class="date-wrapper">
-                            <date-pick
+<!--                          https://github.com/dbrekalo/vue-date-pick/issues/74  -->
+                          <div class="date-wrapper">
+                            <span class="fa fa-calendar calendar-icon"></span>
+                              <date-pick style="width:167px"
                                  v-model="startdate"
```

Comment:
keeping style outside of class is my informal convention to know to revisit when consolidating CSS files, scoped and other inline <style> that is sprawled all over the place, often with unused/stale code; temporary convention during ongoing cleanup